client/src/modules/archived/index.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { Fragment, PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import _map from 'lodash/map';
+
+import Toolbar from 'common/components/Toolbar';
+import { fetchShoppingListsMetaData } from 'modules/shopping-list/model/actions';
+import { getShoppingLists } from 'modules/shopping-list/model/selectors';
+import CardItem from '../dashboard/CardItem';
+import { ListIcon } from 'assets/images/icons';
+
+class Archived extends PureComponent {
```

Comment:
is it only for lists? If yes than why so generic name?

Reply:
It's not only for list. I added archived cohorts section to this component. It displays only message for now